src/inc-lib-init.php

```diff
@@ -1175,6 +1175,16 @@ function lovd_getVariantInfo ($sVariant, $sTranscriptID = '', $bCheckHGVS = fals
                             ' For ' . $sVariant[0] . '. variants, please use a genomic reference sequence.';
                         break;
                 }
+
+            } elseif (!preg_match('/^(N[CGT]|LRG|ENSG)/', $sReferenceType)
```

Comment:
Somehow my earlier addition of NT and NW reference sequences was deleted here. It's now sometimes allowed and sometimes not, which seems inconsistent. It's good that LRG and ENSG got added, was the removal of NT and NW an accident or a deliberate change?